Resources/translations/SonataBlockBundle.en.xliff

```diff
@@ -2,24 +2,24 @@
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
     <file source-language="en" datatype="plaintext" original="file.ext">
         <body>
-            <trans-unit id="sonata.block.container">
-                <source>sonata.block.container</source>
+            <trans-unit id="sonata.block.service.container">
```

Comment:
IMO, just removing the `.service` will solve all our BC break issue. Am I wrong?